sql/src/test/java/io/crate/planner/consumer/ManyTableConsumerTest.java

```diff
@@ -1,351 +0,0 @@
-/*
- * Licensed to Crate under one or more contributor license agreements.
- * See the NOTICE file distributed with this work for additional
- * information regarding copyright ownership.  Crate licenses this file
- * to you under the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.  You may
- * obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
- * implied.  See the License for the specific language governing
- * permissions and limitations under the License.
- *
- * However, if you have executed another commercial license agreement
- * with Crate these terms will supersede the license and you may use the
- * software solely pursuant to the terms of the relevant commercial
- * agreement.
- */
-
-package io.crate.planner.consumer;
-
-import com.carrotsearch.hppc.ObjectIntHashMap;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Sets;
-import io.crate.action.sql.SessionContext;
-import io.crate.analyze.Analysis;
-import io.crate.analyze.MultiSourceSelect;
-import io.crate.analyze.OrderBy;
-import io.crate.analyze.ParameterContext;
-import io.crate.analyze.SelectAnalyzedStatement;
-import io.crate.analyze.TwoTableJoin;
-import io.crate.analyze.relations.JoinPair;
-import io.crate.analyze.symbol.Field;
-import io.crate.analyze.symbol.Literal;
-import io.crate.analyze.symbol.Symbol;
-import io.crate.metadata.OutputName;
-import io.crate.planner.node.dql.join.JoinType;
-import io.crate.sql.parser.SqlParser;
-import io.crate.sql.tree.QualifiedName;
-import io.crate.test.integration.CrateDummyClusterServiceUnitTest;
-import io.crate.testing.SQLExecutor;
-import io.crate.testing.SqlExpressions;
-import io.crate.testing.T3;
-import io.crate.types.DataTypes;
-import org.junit.Before;
-import org.junit.Test;
-
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
-import static io.crate.testing.TestingHelpers.isSQL;
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.nullValue;
-import static org.hamcrest.core.Is.is;
-
-public class ManyTableConsumerTest extends CrateDummyClusterServiceUnitTest {
```

Comment:
Not directly. Some of the tests here were for the re-ordering optimization which is still missing. Other tests were for the field-remapping which was necessary because we were building new AnalyzedRelation instances - this is now no longer the case.

I could probably add a couple more Join Operator unit tests (specially for the Builder part)